### PR TITLE
Evo 1209 plan page

### LINKED ISSUE #1209: "Group Subscription" Management Page for mods and/or users?
A list page with all the subscriptions that a user is signed up for, and all the subscriptions from groups that they moderate.

Allow moderators to change the visibility of a plan there, and link to where it can be edited/updated

Accessible from the user settings

### PR DESCRIPTION
**Pull request by thomasgwatson**
Closes #1209

### CODE REVIEW

File: src/components/PlansTab/PlansTab.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { bgImageStyle } from 'util/index'
+import { capitalize } from 'lodash'
+import { useCurrentUser } from 'hooks/useCurrentUser'
+import { useEnsureGroupPlans } from 'hooks/useEnsureGroupPlans'
+import { useEnsureUserPlans } from 'hooks/useEnsureUserPlans'
+import Pill from 'components/Pill'
```
Comment (tibetsprague) [inline src/components/PlansTab/PlansTab.js:8]: very minor thing but i do still prefer to alphabetize these lists. my flow is 
- alphabetized list of external modules (so lodash would go above react)
- alphabetized list of our code in other folders (so the components would go above hooks, and Icon above Pill)
- Files that are within or underneath the current directory (scss one last)

File: src/components/PlansTab/PlansTab.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    - currentUser
+    - (groupTab) all plans associated with a group
+    - (userTab) all plans for groups the user moderates
+    - (userTab) all plans a user is subscribed to
+
+    What needs to be shown?
+    group tab
+    - A 'Create plan' button to navigate to the plan creation
+    - list of plans associated with group
+      - Highlight any plans the currentUser is linked to
```
Comment (tibetsprague) [inline src/components/PlansTab/PlansTab.js:36]: can a user be part of more than one plan in a group? is there a use case for this?

File: src/routes/FullPageModal/FullPageModal.scss [hunk 3]
```diff
@@ -30,7 +30,7 @@ $size-sidebar-width: 13rem;
   padding-right: 32px;
   .left-sidebar-fixed {
     position: fixed;
-    padding-top: 220px;
+    padding-top: 160px;
```
Comment (tibetsprague) [inline src/routes/FullPageModal/FullPageModal.scss:33]: why this change?

**Review by tibetsprague — COMMENTED** [review]
not quite sure what to do with this, obviously theres much more to do, shall we merge with payment-plans so we can both work on it?

File: src/components/PlansTab/PlansTab.js (hunk 2 above)
Comment (thomasgwatson) [reply src/components/PlansTab/PlansTab.js:36]: No, I don't think so.

File: src/routes/FullPageModal/FullPageModal.scss (hunk 3 above)
Comment (thomasgwatson) [reply src/routes/FullPageModal/FullPageModal.scss:33]: Because I added another listing in the left side, the bottom of the list was getting too low in smaller screens.

File: src/routes/GroupSettings/GroupSettings.js [hunk 4]
```diff
@@ -41,77 +42,90 @@ export default function GroupSettings ({
...
-    ]}
+  return (
+    <FullPageModal
+      goToOnClose={groupUrl(slug)}
+      content={[
+        {
+          name: 'Settings',
+          path: groupUrl(slug, 'settings'),
+          component:
+  <GroupSettingsTab
```
[the remark this answers is not in the document]
Comment (thomasgwatson) [reply src/routes/GroupSettings/GroupSettings.js:53]: Indeed! This is the result of the linter's newer demands. Assuming we are sticking to that where we can?

**Comment by thomasgwatson** [comment]
Overall: based on our design discussions, the layout of this PR will be superseded. I'm suggesting we merge to `payment-plan` and break the restyle out into a separate ticket. That way the none layout aspects of this PR can still be available as a foundation for other tickets

@tibetsprague
